frontend/template-mixins/mixins/template-mixins.js

```diff
@@ -237,6 +237,7 @@ module.exports = function (options) {
         legendClassName: legendClassName,
         role: opts.type === 'radio' ? 'radiogroup' : 'group',
         ariaRequired: opts.type === 'radio',
+        hintHTML: hintHTML ? hintHTML : '',
```

Comment:
I wonder what this used this for?

Reply:
this is good, nhs have added some dynamic content for showing hiddent and reveal text.  This might need to be renamed to matched whatever it is called on the html5 or however it is named in govuk design system

Reply:
it's called detail which is a html5 thing and a component in the design system